infrastructure/docker/celo/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM golang:1.14-alpine AS gobuild
+
+RUN apk add --update --no-cache \
+    g++ \
+    linux-headers \
+    make \
+    git
+
+ENV BIN_PATH=/usr/local/bin
+
+# Clone, compile, and install Celo blockchain v1.2.2
+RUN git clone https://github.com/celo-org/celo-blockchain.git
+WORKDIR /go/celo-blockchain
+RUN git checkout v1.2.2
+RUN make geth-musl
+RUN cp build/bin/geth $BIN_PATH
+
+FROM alpine:3.10
+
+# Volume where chain data are persisted
+VOLUME [$CELO_DATA_DIR]
```

Comment:
Shouldn't this be below `ENV CELO_DATA_DIR=/mnt/data`?

Reply:
Yes, should be! Fixed.